Jenkinsfile.github

```diff
@@ -57,7 +57,62 @@ pipeline {
 
     stages {
 
-        stage('Build & Publish') {
+        stage('Image: Build & Publish') {
+
+            agent {
+                node {
+                    label "metal-gcp-builder"
+                }
+            }
+
+            stages {
+                
+                stage('Build dev image') {
+                  steps {
+                      script {
+                          sh "python3 -m pip install setuptools_scm[toml]"
+                          version = sh(returnStdout: true, script: "python3 -m setuptools_scm | tr -s '-' '~' | tr -s '+' '_' | sed 's/^v//'").trim()
+                          sh "make dev_image"
+                      }
+                  }
+                }
+
+                stage('Publish dev image') {
+                  steps {
+                      script {
+                          // Use setuptools_scm to resolve the version(s) to use for the build.
+                          sh "python3 -m pip install setuptools_scm[toml]"
+                          version = sh(returnStdout: true, script: "python3 -m setuptools_scm | tr -s '-' '~' | tr -s '+' '_' | sed 's/^v//'").trim()
+                          publishCsmDockerImage(image: "${env.NAME}", tag: "${version}-dev", isStable: isStable)
+                      }
+                  }
+                }
+
+                stage('Build prod image') {
+                  steps {
+                      script {
+                          sh "python3 -m pip install setuptools_scm[toml]"
+                          version = sh(returnStdout: true, script: "python3 -m setuptools_scm | tr -s '-' '~' | tr -s '+' '_' | sed 's/^v//'").trim()
+                          sh "make prod_image"
```

Comment:
Maybe `make image` so it uses the "default" image path (emphasizing that the default path is the prod image).